tokio-timer/src/delay_queue.rs

```diff
@@ -501,10 +501,13 @@ impl<T> DelayQueue<T> {
 
         self.slab[key.index].when = when;
 
+        let next_poll = self.wheel.poll_at()
+            .map(|t| self.start + Duration::from_millis(t));
+
         if let Some(ref mut delay) = self.delay {
             debug_assert!(old >= delay.deadline());
```

Comment:
Do we need this _or_ `old` anymore?

Reply:
2fast2furious with my review comments, I guess!

It's probably fine to leave, although maybe a comment on what it's doing would be good?  That's the only thing that catches me off guard with it sticking around but not being directly used.

Reply:
I moved it inside of the `debug_assert!` in 3c2e16edf1caaed8fb4f3d233dbdbfd8121878c7. That way, it's clear that `old` is only necessary to make the assertion, and we don't have to calculate the value of `old` if we're skipping the assertion. :D